File: sbe/binance/stream_1_0/generated-python/spot_stream/BoolEnum.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple, Set
from io import BytesIO

class BoolEnum:
    """Enum values."""
    FALSE = 0
    TRUE = 1
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/binance/stream_1_0/generated-php/spot_stream/BoolEnum.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class BoolEnum
{
    public const FALSE = 0;
    public const TRUE = 1;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}
